SwipeActions.js

```diff
@@ -17,13 +17,13 @@ import React, {
 } from 'react-native';
 
 const START_SWIPE_THRESHOLD = 10.0;
-const ACTION_BUTTON_WIDTH = 90;
+const ACTION_BUTTON_DEFAULT_WIDTH = 90;
 const ANIMATION_DURATION_MS = 50;
 const CLOSE_SWIPE_ACTIONS_EVENT = 'closeSwipeActions';
 
 /**
- * `actions` prop is an array of objects with `text`, `backgroundColor`,
- * and `onPress` fields.
+ * `actions` prop is an array of objects with `text`, `containerStyle`,
```

Comment:
maybe call `containerStyle` just `style` since it's implied that the style applies to the container component